topics/transcriptomics/tutorials/scrna-raceid/tutorial.md

```diff
@@ -0,0 +1,690 @@
+
+**StemID** is a tool (part of the **RaceID** package) that makes use of this topology to derive a hierarchy of these cell types by constructing a cell lineage tree, rooted at the cluster(s) believed to best describe multipotent progenitor stem cells, and terminating at the clusters which describe more mature cell types.
+
+
+## Computing the Lineage Tree
+
+> ### {% icon hands_on %} Hands-on
+>
+> 1. **Lineage computation using StemID** {% icon tool %} with the following parameters:
+>    - {% icon param-file %} *"Input RDS"*: `outrdat` (output of **Clustering using RaceID** {% icon tool %})
+>    - In *"Compute transcriptome entropy of each cell"*:
+>        - *"Use Defaults?"*: `Yes`
+>    - In *"Compute Cell Projections for Randomized Background Distribution"*:
+>        - *"Use Defaults?"*: `Yes`
+>    - In *"StemID2 Lineage Graph"*:
+>        - *"Use Defaults?"*: `Yes`
+>
+{: .hands_on}
+
+![Lineage Computation Plots]({{site.baseurl}}{% link topics/transcriptomics/images/raceid_stemid_lineage.png %} "Lineage Tree and Branches of significance.")
+
+The first (top-left) plot shows a minimum spanning tree that summarises the most likely connections between clusters. The second plot (top-right) elucidates this lineage hierarchy by projecting the cells along the links given by StemID, ordering the cells along each link in such a way to suggest a time series, or *pseudotime* analysis of each link. This is important, because by ordering the cells by lineage pseudotime we can trace the up or down regulation of a gene as discrete time points. The interval between each timepoint is of course hard to accurately determine, but the order of events is still important.
+
+The third (bottom-left) lends a degree of significance between clusters, where the colour indicates the level of significance between clusters, and the width indicates the link score computed by **StemID**. The thick red link between clusters 2-4, 3-5, and 3-1-6 have therefore a high level of significance attached to each of these.
+
+This is clarified slightly better with the heatmap (bottom-right) that shows the link scores between each cluster-cluster pair. To see how the link score is actually calculated, the image below provides some context:
+
+> ### {% icon question %} Questions
+>
+> Based on the above plots, which cluster is likely to be the main progenitor of the others?
+> 
+> > ### {% icon solution %} Solution
+> >
+> > On first glance, it appears that `c2` would be the progenitor, giving rise to; mature type `c11`, `c4`, and `c9`; `c3` which is the progenitor to `c5` and `c1`; `c8`, which gives rise to `c12` that in turn generates `c10`.
+> > `{2, {8 {12 {10}}, 11, 4, 9, 3 {5, 1 { 6 } } } }` (in JSON format)
+> > 
+> > This is actually not the case, and we will see why with the following images.
```

Comment:
while I like the Q, this sentence "This is actually not the case, and we will see why with the following images." which serves as motivation to the next images is hidden and do not motivate people that have not read the Q

So I would take this sentence out of the box.